frontend/src/store/modules/interactionPartners.js

```diff
@@ -15,7 +15,10 @@ const data = {
 const getters = {
   component: state => state.interactionPartners.component || {},
   reactions: state => state.interactionPartners.reactions || [],
-  title: (state, _getters) => (_getters.component.type === 'metabolite' ? chemicalName(_getters.component.name) : _getters.component.name),
+  title: (state, _getters) => (_getters.component.type === 'metabolite'
+    ? chemicalName(_getters.component.name || _getters.component.id)
```

Comment:
The interpretation of some of the chemical names are wrong, e.g. for 
`m02043c,h2s2o3  /explore/Human-GEM/interaction-partners/m02043c`. I think it should be H<sub>2</sub>S<sub>2</sub>O<sub>3</sub>.

Reply:
Good that you noticed this. Could you please open up a new issue for this? It's not directly related the purpose of this issue and PR.

Reply:
There are more problems with the function `chemicalName`.  For example, `/explore/Human-GEM/interaction-partners/m02046c`, it should be HCO<sub>3</sub><sup>-</sup>, not HCO<sub>3</sub><sub>-</sub>. I think we should create a separate issue for that.

Reply:
Thank you for creating the new issue!